demo/shaders/glsl/mesh_pbr_uniform.glsl

```diff
@@ -48,6 +48,16 @@ struct ShadowMapCubeData {
     float cube_map_projection_far_z;
 };
 
+const int MAX_POINT_LIGHTS = 16;
+const int MAX_DIRECTIONAL_LIGHTS = 16;
+const int MAX_SPOT_LIGHTS = 16;
+
+const int MAX_SHADOW_MAPS_2D = MAX_DIRECTIONAL_LIGHTS + MAX_SPOT_LIGHTS;
+const int SHADOW_MAP_2D_ARRAY_LEN = 32; // TODO(dvd): Shader processor can't handle const math in array indices.
```

Comment:
I wonder if these show up in SPIR-V.. if they do, then we might could get the value directly from there. (future work, possibly). spirv-cross might even have something that could help